tests/test_end_to_end.py

```diff
@@ -11,4 +13,7 @@ def test_train(self, cfg):
         train(cfg)
 
     def test_predict(self, cfg):
-        predict(cfg)
+        sdss = instantiate(cfg.predict.dataset)
+        prepare_img = prepare_image(sdss[0]["image"], cfg.predict.device)
+        prepare_bg = prepare_image(sdss[0]["background"], cfg.predict.device)
+        predict(cfg, prepare_img, prepare_bg)
```

Comment:
It'd be good to avoid duplicating these 4 lines of code (both here and in main.py). Perhaps make a function called `predict_sdss(cfg)` in `predict.py` that is called here and in main.py, so that code isn't duplicated